src/arviz_plots/plots/utils.py

```diff
@@ -19,3 +19,60 @@ def filter_aes(pc, aes_map, artist, sample_dims):
     _, all_loop_dims = pc.update_aes(ignore_aes=ignore_aes)
     artist_dims = [dim for dim in sample_dims if dim not in all_loop_dims]
     return artist_dims, artist_aes, ignore_aes
+
+
+def get_size_of_var(group, compact=False):
+    """Get the size of the variables in a group."""
+    coords = group.coords._names - {"chain", "draw"}  # pylint: disable=protected-access
```

Comment:
I think this could/should? be 

```suggestion
    coords = set(group.sizes) - set(sample_dims)
```